internal/characters/dori/burst.go

```diff
@@ -0,0 +1,109 @@
+package dori
+
+import (
+	"math"
+
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/avatar"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/event"
+	"github.com/genshinsim/gcsim/pkg/core/player"
+)
+
+var burstFrames []int
+
+const (
+	burstHitmark    = 28
+	burstHealPeriod = 120
```

Comment:
if you're gonna have a variable for the heal tick rate, then add one for the dmg tick rate as well, because it feels weird to only have it for one of them. (or remove it and use 120 below)